Content.Server/Power/Components/ApcComponent.cs

```diff
@@ -26,6 +26,7 @@ public sealed class ApcComponent : BaseApcNetComponent
 
     [ViewVariables]
     public bool MainBreakerEnabled = true;
+    public bool HasAccess = false;
```

Comment:
Doesnt this make the UI dependent on whoever opens it last? (like if a passenger leaves it open and then an engineer opens it, the passenger will have the button colored or some weird combination like that) you can test with two clients

Reply:
Tha access bool gets changed every time you open the UI so it will never keep its state the same for different users. I understand maybe a server side bool isn’t best but couldn’t find any examples of UI that changes it’s view depended on the users access (IDCardConsole checks entered ID not users ID)

For the buttons. I agree but really don’t want to do it right now, it’s a little complex since the whole system works on an assumed toggle message and honestly I wanted to get this facelift off the ground first and tackle that later along with power channels.